test/testenv/verificationutils.go

```diff
@@ -676,3 +676,41 @@ func VerifyAppsDownloadedByInitContainer(deployment *Deployment, testenvInstance
 		}
 	}
 }
+
+// VerifyBundlePush verify that bundle push was pushed on all indexers
+func VerifyBundlePush(deployment *Deployment, testenvInstance *TestEnv, ns string, podName string, replicas int, previousBundleHash string) {
+	gomega.Eventually(func() bool {
+		// Get Bundle status and check that each pod has successfully deployed the latest bundle
+		clusterMasterBundleStatus := ClusterMasterBundlePushstatus(deployment, previousBundleHash)
+		if len(clusterMasterBundleStatus) < replicas {
+			testenvInstance.Log.Info("Bundle push on Pod not complete on all pods", "Pod with bundle push", clusterMasterBundleStatus)
+			return false
+		}
+		allPodNames := DumpGetPods(testenvInstance.GetName())
+		for _, podName := range allPodNames {
+			if strings.Contains(podName, "-indexer-") {
+				if _, present := clusterMasterBundleStatus[podName]; present {
+					if clusterMasterBundleStatus[podName] != "Up" {
+						testenvInstance.Log.Info("Bundle push on Pod not complete", "Pod Name", podName, "Status", clusterMasterBundleStatus[podName])
+						return false
+					}
+				} else {
+					testenvInstance.Log.Info("Bundle push on Pod not found on pod", "Podname", podName)
```

Comment:
nit: maybe "Bundle push not found on pod:" would be more clear than "Bundle push on Pod not found on pod"